### PR TITLE
Fix build number prompt for multiple target

### PR DESCRIPTION
**Pull request by AncAinu**

### CODE REVIEW

**Comment by AncAinu** [comment]
I see Travis isn't happy about:
```
Fastfile:104:3: C: Style/RedundantReturn: Redundant return detected.
  return ENV.include?('TAB_TARGET_FOR_BUILD_NUMBER') ? get_version_number(target: ENV['TAB_TARGET_FOR_BUILD_NUMBER']) : get_version_number
  ^^^^^^
```
Probably because I am not a ruby dev, although I test locally and it works

File: Fastfile [hunk 1]
```diff
@@ -100,6 +100,10 @@ def _build_number
   end
 end
 
+def _get_project_build_number
+  return ENV.include?('TAB_TARGET_FOR_BUILD_NUMBER') ? get_version_number(target: ENV['TAB_TARGET_FOR_BUILD_NUMBER']) : get_version_number
```
Comment (zacoid55) [inline Fastfile]: I can see RuboCop complaining about the length of this line, could you spread it across a couple of lines?

**Comment by KaneCheshire** [comment]
@AncAinu Rubocop is just complaining that you don't need the `return` because the last line in a function always returns the value in Ruby. If you remove the `return` it should still work fine :)

File: Fastfile (hunk 1 above)
Comment (AncAinu) [reply Fastfile]: @zacoid55  should I do a classic `if else` ? Sorry not a ruby pro at all

Comment (KaneCheshire) [inline Fastfile]: I wonder if instead of `TAB_TARGET_FOR_BUILD_NUMBER` we have something more generic like `TAB_PRIMARY_TARGET`?

**Comment by AncAinu** [comment]
Rubocop is happy now

File: Fastfile (hunk 1 above)
Comment (AncAinu) [reply Fastfile]: As you wish really, although I don't see the need.
> No need to try solving a problem which doesn't exist

**Review by KaneCheshire — APPROVED** [review]

**Pull request merged**